tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__Handler.java

```diff
@@ -52,7 +53,11 @@
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (channelUID.getId().equals(CHANNEL_1)) {
+        if (CHANNEL_1.equals(channelUID.getId())) {
```

Comment:
`channelUID.getId()` is guaranteed to be non-null, so I don't think this change is necessary - imho it actually makes the code a bit harder to read, because what one really does here is to do a "switch" on the channel ID.

Reply:
@martinvw requested in his review for openhab/#2990 to change the order and as it was not originally my code I changed it here.
Happy to change it at valloxmv binding the other way round

Reply:
So let's hear @martinvw's response to my comment :-)

Reply:
If we want a switch then we should make it a switch ;-)

Your comment is fair but the general idea to have literals on the left is quite well spread and a sane programming practice. In this case it might be without any risk but the example which it sets is not a good default :-)

Reply:
I agree that it is in general the safer way, but it makes reading less fluent (as you would normally say "if the channel is X" and not "if X is the channel".
But ok, I do not care too much - if you feel it is important as devs might otherwise do mistakes in other parts of the code, let's do it your way :-)